benchmark/CMap.hs

```diff
@@ -42,16 +52,35 @@ tenLookups tmap = (lp, lp, lp, lp, lp, lp, lp, lp)
     lp :: forall (a::Nat). Typeable a => Proxy a
     lp = fromJust $ lookup tmap
 
+inserts :: forall a . (KnownNat a)
+        => TypeRepMap (Proxy :: Nat -> *)
+        -> Int
+        -> Proxy (a :: Nat)
+        -> TypeRepMap (Proxy :: Nat -> *)
+inserts !c 0 _ = c
+inserts !c n x = inserts
+   (insert x c)
+   (n-1)
+   (Proxy :: Proxy (a+1))
+
+updates :: forall a . (KnownNat a)
+        => TypeRepMap (Proxy :: Nat -> *)
+        -> Int
+        -> Proxy (a :: Nat)
+        -> TypeRepMap (Proxy :: Nat -> *)
+updates !c 0 _ = c
+updates !c n x = inserts
+   (insert x c)
+   (n-1)
+   (Proxy :: Proxy (a+1))
+
+mkSmallMap :: IO (TypeRepMap (Proxy :: Nat -> *))
+mkSmallMap = pure $ buildBigMap 10 (Proxy :: Proxy 0) empty
+
 -- TypeRepMap of 10000 elements
-bigMap :: TypeRepMap (Proxy :: Nat -> *)
-bigMap = buildBigMap 10000 (Proxy :: Proxy 0) empty
+mkBigMap :: IO (TypeRepMap (Proxy :: Nat -> *))
+mkBigMap = pure $ buildBigMap 10000 (Proxy :: Proxy 0) empty
```

Comment:
I guess this can be a separate function which takes `Int`. Then `mkSmallMap`and `mkBigMap` can be implemented using that.

Reply:
ok